portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,34 @@
-#content {
+body{
+    background-color: black;
+}
+.name{
+    position: absolute;
+    top: 20%;
+    left: 50%;
+    text-align: center;
+    width: 450px;
+    color: white;
+    /* border-spacing: 1em .5em; */
+    padding: 30px;
+    border: 5px solid white; 
+    transform: translate(-50%, -50%);
+
+}
+.soon{
```

Comment:
similar with this naming.

Reply:
changed the naming to "websiteinprogress"